src/main/java/org/candlepin/resource/CloudRegistrationResource.java

```diff
@@ -288,4 +298,23 @@ private void validateCloudAuthenticationResult(CloudAuthenticationResult result)
             throw new NotAuthorizedException(errmsg);
         }
     }
+
+
+    @Override
+    @Transactional
+    public void deleteAnonymousConsumersByAccountId(String cloudAccountId) {
+        Principal principal = this.principalProvider.get();
+        if (!principal.hasFullAccess()) {
+            throw new NotAuthorizedException(
+                i18n.tr("This endpoint requires access that the user does not have"));
+        }
```

Comment:
This is redundant; if an endpoint does not have a `@SecurityHole` annotation on it, and there is also no `@Verify` annotation on any of the arguments, then the default authorization filter is used (`SuperAdminAuthorizationFilter`) which is already doing this check